### PR TITLE
[File System Access API article] Add directory serialization example

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add directory serialization example.

### CODE REVIEW

File: src/site/content/en/blog/file-system-access/index.md [hunk 1]
```diff
@@ -223,47 +223,65 @@ file at a specific position, or resize the file.
...
+    await set("directory", directoryHandle);
+    pre2.textContent = `Stored directory handle for "${directoryHandle.name}" in IndexedDB.`;
   } catch (error) {
     alert(error.name, error.message);
   }
 });
 ```
 
+### Stored file or directory handles and permissions
+
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:284]: It looks like some text below here might need to be updated as well. Line 286, for example, should be 'has granted permission to the file or directoruy...'

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/file-system-access/index.md (hunk 1 above)
Comment (tomayac) [reply src/site/content/en/blog/file-system-access/index.md:284]: Good point. Added in https://github.com/GoogleChrome/web.dev/pull/4698/commits/49dc0144e595c60b43b26991f6d526820866338d.

File: src/site/content/en/blog/file-system-access/index.md [hunk 2]
```diff
@@ -223,50 +223,71 @@ file at a specific position, or resize the file.
...
-### Storing file handles in IndexedDB
+### Storing file handles or directory handles in IndexedDB
 
-File handles are serializable, which means that you can save a file handle to IndexedDB, or call
-`postMessage()` to send them between the same top-level origin.
+File handles and directory handles are serializable, which means that you can save a file or
+directory handle to IndexedDB, or call `postMessage()` to send them between the same top-level origin.
 
-Saving file handles to IndexedDB means that you can store state, or remember which files a user was
+Saving file or directory handles to IndexedDB means that you can store state, or remember which files
```
Comment (mihajlija) [inline src/site/content/en/blog/file-system-access/index.md:231]:
```suggestion
Saving files or directory handles to IndexedDB means that you can store state, or remember which files
```

File: src/site/content/en/blog/file-system-access/index.md [hunk 3]
```diff
@@ -223,50 +223,71 @@ file at a specific position, or resize the file.
...
+or directories a user was
 working on. This makes it possible to keep a list of recently opened or edited files, offer to
-re-open the last file when the app is opened, etc. In the text editor, I store a list of the five
+re-open the last file when the app is opened, restore the previous working directory, etc.
```
Comment (mihajlija) [inline src/site/content/en/blog/file-system-access/index.md:234]:
```suggestion
re-open the last file when the app is opened, restore the previous working directory, and more.
```

File: src/site/content/en/blog/file-system-access/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/file-system-access/index.md:231]: Not in this case. It's to be read as "file handles or directory handles".

**Review by mihajlija — APPROVED** [review]

**Pull request merged**